tests/test_key_revocation.py

```diff
@@ -385,9 +388,6 @@ def test_root_key_revocation(self):
       for mirror_exception in exception.mirror_errors.values():
         self.assertTrue(isinstance(mirror_exception, tuf.BadSignatureError))
 
-    # Load the previous Root signing key so that the the client can update
-    # successfully.
-    repository.root.load_signing_key(self.role_keys['root']['private'])
```

Comment:
Haven't finished looking at the other changes carefully, but why was this moved to lines 370 - 372?  The purpose of this test condition (starting on line 388) is to verify that the client rejects the root file because it doesn't trust any of its keys (the root key hasn't signed it yet).